cards/eyeddx

```diff
@@ -0,0 +1,31 @@
+title: See the Unseen: Use the Rule of 2s to Never Miss Another Ocular Emergency
```

Comment:
I would use a less editorial title. Something like "Rule of 2s for Ocular Emergencies"